pkg/mount/README.md

```diff
@@ -0,0 +1,34 @@
+mount
+=====
+
+Mount block devices for use in Linuxkit
+
+**NOTE: Block devices may only be mounted in `/var` unless you have explicitly added an additional bind mount**
+
+## Usage
+
+This packages supports two modes of use:
+
+### Automount
+
+```
+onboot:
+  name: mount
+  image: linuxkit/format:${hash}
+  command: ["/usr/bin/mountie", "/var/lib/docker"]
+```
+
+In this mode of operation, the first linux partition on the first block device is mounted to the mountpoint provided.
+Multiple mount points are supported. In the case of multiple mount points, it will attempt to mount the first available linux partition on the remaining block devices (i.e those that have not already been used by previous mounts).
+
+### Manual mount
+
+```
+onboot:
+  name: format
+  image: linuxkit/format:${hash}
+  command: ["/usr/bin/format", "/var/lib/docker:/dev/vda1", "/var/foo:/dev/vdb1" ]
```

Comment:
@dave-tucker I liked this identical format in #2173 but consensus was one format per container. I think the thrust was that having multiple `onboot` containers, each with the same image but different options to format or mount a different disk, is a near-zero-cost overhead. 

I think my own leaning was traditional. We do run `mount` once per disk/mountpoint, which would be equivalent to one container. But when we have all of the mount points in a config (`fstab`), a single command mounts them all, which is the equivalent of a single container.